### DISCUSSION TITLE
F-statistic

### DISCUSSION THREAD

**Discussion by neuroGB** [MRCIEU/TwoSampleMR#413] [General]
Is there a way to calculate and obtain F-statistics for IVs within the R package?

**Comment by remlapmot** [comment]
Please see the **MVMR** <https://wspiller.github.io/MVMR/index.html> package and its `strength_mvmr()` function

<https://wspiller.github.io/MVMR/articles/MVMR.html#step-3-test-for-weak-instruments-1>
<https://wspiller.github.io/MVMR/reference/strength_mvmr.html>

which implements the methods in Eleanor's paper for genotype summary level data <https://doi.org/10.1002/sim.9133>.

**Reply by neuroGB** [reply]
So there's no way to do it within the TwoSampleMR package...?
Also, do we need the SummaryStats for the exposure, too, to calculate the F stat? I believed not...

Many thanks, anyway!
